Sources/Protocols/ReuseViewItem/CollectionViewReusableViewItem.swift

```diff
@@ -19,11 +20,20 @@ public enum ReusableViewType {
     }
 }
 
-public protocol CollectionViewReusableViewItem {
+public protocol CollectionViewReusableViewItem: CollectionViewSiblingCellItem {
     
     var type: ReusableViewType { get set }
+
+    var classType: UICollectionReusableView.Type { get }
```

Comment:
You should use something like `ReuseType` here instead of just `classType` because there are more cases for reusable view registration then just a class